aviary/docs/examples/additional_flight_phases.ipynb

```diff
@@ -52,168 +52,147 @@
-    "            'initial_mach': (0.74, 'unitless'),\n",
-    "            'final_mach': (0.74, 'unitless'),\n",
+    "            'mach_optimize': False,\n",
+    "            'mach_polynomial_order': 1,\n",
+    "            'mach_initial': (0.74, 'unitless'),\n",
+    "            'mach_final': (0.74, 'unitless'),\n",
     "            'mach_bounds': ((0.72, 0.76), 'unitless'),\n",
-    "            'initial_altitude': (33000.0, 'ft'),\n",
-    "            'final_altitude': (33000.0, 'ft'),\n",
+    "            'altitude_optimize': False,\n",
+    "            'altitude_polynomial_order': 1,\n",
+    "            'altitude_initial': (33000.0, 'ft'),\n",
+    "            'altitude_final': (33000.0, 'ft'),\n",
     "            'altitude_bounds': ((32500.0, 33500.0), 'ft'),\n",
     "            'throttle_enforcement': 'boundary_constraint',\n",
-    "            'fix_initial': False,\n",
-    "            'constrain_final': False,\n",
-    "            'fix_duration': False,\n",
-    "            'initial_bounds': ((54.0, 162.0), 'min'),\n",
-    "            'duration_bounds': ((24.0, 72.0), 'min'),\n",
+    "            'time_initial_bounds': ((54.0, 162.0), 'min'),\n",
+    "            'time_duration_bounds': ((24.0, 72.0), 'min'),\n",
     "        },\n",
     "        'initial_guesses': {'time': ([108, 48], 'min')},\n",
     "    },\n",
     "    'climb_3': {\n",
     "        'subsystem_options': {'core_aerodynamics': {'method': 'computed'}},\n",
     "        'user_options': {\n",
-    "            'optimize_mach': False,\n",
-    "            'optimize_altitude': False,\n",
-    "            'polynomial_control_order': 1,\n",
     "            'num_segments': 3,\n",
     "            'order': 3,\n",
-    "            'solve_for_distance': False,\n",
-    "            'initial_mach': (0.74, 'unitless'),\n",
-    "            'final_mach': (0.76, 'unitless'),\n",
+    "            'mach_optimize': False,\n",
+    "            'mach_polynomial_order': 1,\n",
+    "            'mach_initial': (0.74, 'unitless'),\n",
+    "            'mach_final': (0.76, 'unitless'),\n",
     "            'mach_bounds': ((0.72, 0.78), 'unitless'),\n",
-    "            'initial_altitude': (33000.0, 'ft'),\n",
-    "            'final_altitude': (34500.0, 'ft'),\n",
+    "            'altitude_optimize': False,\n",
+    "            'altitude_polynomial_order': 1,\n",
+    "            'altitude_initial': (33000.0, 'ft'),\n",
+    "            'altitude_final': (34500.0, 'ft'),\n",
     "            'altitude_bounds': ((32500.0, 35000.0), 'ft'),\n",
     "            'throttle_enforcement': 'boundary_constraint',\n",
-    "            'fix_initial': False,\n",
-    "            'constrain_final': False,\n",
-    "            'fix_duration': False,\n",
-    "            'initial_bounds': ((78.0, 234.0), 'min'),\n",
-    "            'duration_bounds': ((7.0, 21.0), 'min'),\n",
+    "            'time_initial_bounds': ((78.0, 234.0), 'min'),\n",
```

Comment:
this is a really nice change so we can easily tell that initial and duration bounds refers to time.